lesson_03/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/AlekseiKanash/golang-course/lesson_03/average"
+	"github.com/AlekseiKanash/golang-course/lesson_03/maps"
+	"github.com/AlekseiKanash/golang-course/lesson_03/strings"
+)
+
+func main() {
+	average.TestSlice()
```

Comment:
I guess it will be useful for you https://pkg.go.dev/testing. We will study it later, but you can use now=)

Reply:
I would prefer to wait until we will study it, just have not enough time right now.